app/services/db/doc_auth_log/blanket_drop_off_rates_per_sp_all_time.rb

```diff
@@ -12,7 +12,8 @@ def call(title, issuer)
       def verified_user_counts_query
         <<~SQL
           select count(*) from identities
-          where ial>=2 and service_provider = '#{@issuer}'
+          where ial>=2 and service_provider = '#{@issuer}' and
+          user_id in (select user_id from profiles)
```

Comment:
My first thought is that it would be better to write this as an explicit JOIN but I tested it out in psql locally, and the query planner did the exact same thing for both

Reply:
cool